src/Schema/Event.ts

```diff
@@ -93,3 +105,38 @@ export interface SuccessfullyLoggedIn {
   auth_redirect: string
   service: AuthService
 }
+
+/**
+ * A user taps a rail, card, or other grouping of entities on iOS
+ *
+ * This schema describes events sent to Segment from [[tappedEntityGroup]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedEntityGroup"
+ *    context_module: "newWorksByArtistsYouFollowRail"
+ *    context_screen_owner_type: Home
+ *    destination_screen_owner_type: Artwork
+ *    destination_screen_owner_id: "5e9a7a238483bf000e2c4c5e"
+ *    destination_screen_owner_slug: "romain-jacquet-lagreze-makeshift-garden-hong-kong"
+ *    horizontal_slide_position: 1
+ *    module_height: "single"
+ *    type: "thumbnail"
+ *  }
+ * ```
+ */
+export interface tappedEntityGroup {
+  action: ActionType.tappedEntityGroup
+  context_module: ContextModule
+  context_screen_owner_type: ScreenOwnerType
+  context_screen_owner_id?: string
+  context_screen_owner_slug?: string
+  destination_screen_owner_type: ScreenOwnerType
+  destination_screen_owner_id?: string
+  destination_screen_owner_slug?: string
+  freshness?: number
+  horizontal_slide_position?: number
```

Comment:
for example: 
<img width="454" alt="Screen Shot 2020-04-23 at 3 06 55 PM" src="https://user-images.githubusercontent.com/26879583/80140287-b99d8f00-8575-11ea-86b7-2eaff269146d.png">